README.md

```diff
@@ -2,11 +2,10 @@
 
 [![GitHub Build Status](https://github.com/cisagov/cool-images-vmimport/workflows/build/badge.svg)](https://github.com/cisagov/cool-images-vmimport/actions)
 
-Terraform code to create resources needed to use the
-[AWS VM Import/Export feature](https://docs.aws.amazon.com/vm-import/latest/userguide/what-is-vmimport.html)
-in the Images (Production) and Images (Staging) accounts in the COOL. This
-includes the `vmimport` service role and `Images-VMImportExportAccess` role for
-each account.
+Terraform code to create resources needed to use the [AWS VM Import/Export
+feature](https://docs.aws.amazon.com/vm-import/latest/userguide/what-is-vmimport.html)
+in the Images account in the COOL. This includes the `vmimport` service role and
+`Images-VMImportExportAccess` role for each account.
```

Comment:
I'd like to reword this while we're here since the "for each account" no longer applies. It could benefit from making it clear that `vmimport` is the service role needed on the AWS side for VM import/export and `Images-VMImportExportAccess` provides a role to manage VM import/export tasks.

Reply:
I think simply removing the "for each account" part should suffice (see https://github.com/cisagov/cool-images-vmimport/pull/33/commits/b2ab96c923c80f0f6840dee185603a4fb0708347), since the roles are clearly explained in the sentences that follow that line.

Reply:
Yep that's what I get for not expanding the hidden parts of the file in the GitHub UI.